lazyflow/operators/valueProviders.py

```diff
@@ -522,8 +529,10 @@ def setupOutputs(self):
         pass
 
     def execute(self, slot, subindex, roi, result):
-        result[...] = 0
-        return result
+        raise MissingDataAccessError(
```

Comment:
The error message could be moved into `MissingDataAccessError.__init__()`.

Also, the operator name is slightly more general than the exception message. Maybe rename it to something like `OpMissingDataSource`?